x-pack/plugins/secops/public/pages/hosts/host_details.tsx

```diff
@@ -61,6 +67,51 @@ export const HostDetails = pure<HostComponentProps>(({ match: { params: { hostId
                   )}
                 </HostsQuery>
               </EuiFlexGroup>
+              <AuthenticationsQuery
+                sourceId="default"
+                startDate={from}
+                endDate={to}
+                poll={poll}
+                filterQuery={getFilterQuery(hostId)}
```

Comment:
It's paying interest already -- good to see the `ESTermQuery` shared type up front. Thanks again 👍